### PR TITLE
Add example akka config to avoid metadata timeout issue

### PR DESCRIPTION
**Pull request by EvanTheB**
Add akka http request-timeout idle-timeout examples in the config
To allow users to get metadata results from large workflows

Also delete the now duplicated cromwell.examples.conf file
And delete the backends section which has been split into
separate files

https://gatkforums.broadinstitute.org/wdl/discussion/10209/retrieving-metadata-for-large-workflows

https://github.com/broadinstitute/cromwell/issues/2519

### CODE REVIEW

**Comment by EvanTheB** [comment]
The file mentions:
`and you can see it defined in references.conf.`

What is references.conf?

**Comment by EvanTheB** [comment]
Also is there any way to actually enumerate all the available settings? Probably this would be too many, and modifying some of them would break the system. Still it is nice to have, and I have tried searching for this 'inbuilt' defaults file in vain. I guess it is scattered across many java files.

**Comment by salonishah11** [comment]
@EvanTheB `reference.conf` is the configuration file used by Cromwell.
> Also is there any way to actually enumerate all the available settings?

I am not sure how we can do that. But you are right, it might be a lot of effort to restructure/modify those files to be able to enumerate it.

File: cromwell.example.backends/cromwell.examples.conf [hunk 1]
```diff
@@ -12,6 +19,16 @@ webservice {
 
 akka {
   # Optionally set / override any akka settings
+  http {
+    server {
+      # Increasing these timeouts allow rest api responses for very large jobs
+      # to be returned to the user. When the timeout is reached the server would respond
+      # `The server was not able to produce a timely response to your request.`
+      # https://gatkforums.broadinstitute.org/wdl/discussion/10209/retrieving-metadata-for-large-workflows
+      # request-timeout = 300s
```
Comment (danbills) [inline cromwell.example.backends/cromwell.examples.conf]: I would prefer to list the default, which I believe is 20s

```suggestion
      # request-timeout = 20s
```

**Comment by danbills** [comment]
@EvanTheB references.conf should be "reference.conf` and it is the default settings for all these options, [seen here](https://github.com/broadinstitute/cromwell/blob/1ebf35417c879cf7ed83fd122564613d9d4d86c8/core/src/main/resources/reference.conf).

There are likely ways to enumerate the settings we are overriding, but the default settings that things like the DB connection Pool are legion and I don't know how to list them.

**Comment by EvanTheB** [comment]
thanks for the reviews

**Pull request merged**